src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.cs

```diff
@@ -500,6 +500,7 @@ public void AddColumn(string columnName, string columnText, int columnWidth, Col
 		public void InsertNumFramesExternal() => InsertNumFramesMenuItem_Click(null, null);
 		public void DeleteFramesExternal() => DeleteFramesMenuItem_Click(null, null);
 		public void CloneFramesExternal() => CloneFramesMenuItem_Click(null, null);
+		public void CloneFramesXTimesExternal() => CloneFramesXTimesMenuItem_Click(null, null);

```

Comment:
This isn't necessary, it's only if we need to run this functionality elsewhere, such as lua

Reply:
Isn't it used for hotkeys?

Reply:
that too, but none of that is wired up in this PR, so I don't think this line should be added, until it is needed